### ISSUE TITLE
UI not loading properly

### ISSUE THREAD

**Issue by anupriya17** [daptin/daptin#95]
Trying to get it running but UI is not accessible. Tried the kubernetes example everything is good, created tables and able to run api.

But the dashboard is not loading properly  noticed the ```content-type``` header is missing in the respose. 

The output in browser as plain text i think instead of html

```
<!DOCTYPE html><html><head><title>DadaDash</title><meta charset=utf-8><meta name=description content="A Daptin Dashboard"><meta name=format-detection content="telephone=no"><meta name=msapplication-tap-highlight content=no><meta name=viewport content="user-scalable=no,initial-scale=1,maximum-scale=1,minimum-scale=1,width=device-width"><link rel=icon type=image/png href=statics/app-logo-128x128.png><link rel=icon type=image/png sizes=16x16 href=statics/icons/favicon-16x16.png><link rel=icon type=image/png sizes=32x32 href=statics/icons/favicon-32x32.png><link rel=icon type=image/png sizes=96x96 href=statics/icons/favicon-96x96.png><link rel=icon type=image/ico href=statics/icons/favicon.ico><link href=css/app.6447ae72.css rel=stylesheet></head><body><div id=q-app></div><script src=js/vendor.61728137.js></script><script src=js/app.c4afe1b5.js></script><script src=js/runtime.00d6c142.js></script></body></html>
```

**Comment by artpar** [comment]
Hi,

Are you able to access the dashboard on running an independent docker container (without k8s)

```docker run -p 8877:8080 daptin/daptin```

Then check if dashboard opens up for you at http://localhost:8877

I do notice what you mentioned about the missing `content-type` header. Let me patch that up.

**Comment by anupriya17** [comment]
Thanks for quick response. I didn't try docker.  I don't have access to the terminal. I'll try on other system

**Comment by anupriya17** [comment]
This worked and able to access the dashboard
```
kubectl -n dev port-forward deployment/daptin 8080:8080
```

But i notice the content-type header missing for main route:

```
Access-Control-Allow-Credentials: true
Access-Control-Allow-Headers: Content-Type,Authorization,X-Requested-With,Accept-Language,Access-Control-Allow-Origin
Access-Control-Allow-Methods: POST,GET,DELETE,PUT,OPTIONS,PATCH
Content-Encoding: gzip
Content-Length: 428
Date: Sun, 06 Sep 2020 13:12:33 GMT
Vary: Accept-Encoding
```

**Comment by artpar** [comment]
Yes seems the content-type header is missing for static files. Will add it.

**Comment by artpar** [comment]
Pushed a fix in d23c10d333c73b8039ed3b297cea3d16a70ed6b0. You can try out the automated build from `daptin/daptin:travis`. Will update the latest image once this is built.
